.github/workflows/ci.yml

```diff
@@ -12,28 +12,58 @@ jobs:
   build:
     strategy:
       matrix:
-        python: ["3.6", "3.7"]
+        python: ["3.6", "3.7", "3.8"]
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v1
-      - name: Setup
+      - name: Conda Bootstrap
+        uses: goanpeca/setup-miniconda@v1
+        with:
+          auto-update-conda: true
+          python-version: ${{ matrix.python }}
+      - name: Conda Config
+        run: |
+          conda config --add channels conda-forge
+          conda config --add channels numba
+          conda config --set channel_priority strict
+          conda config --set always_yes yes
+          conda config --set changeps1 no
+          conda config --set pip_interop_enabled True
+      - name: Conda Install
+        shell: bash -l {0}
+        run: conda install --yes shellcheck --file=requirements.txt --file=test-requirements.txt
+      - name: Pip Install
+        shell: bash -l {0}
+        run: pip install --no-deps -e .[testing]
+      - name: MyPy
+        shell: bash -l {0}
+        run: mypy .
+      - name: PyTest
+        shell: bash -l {0}
+        run: pytest
+      - name: Black
+        shell: bash -l {0}
+        run: black --check .
+      - name: ISort
+        shell: bash -l {0}
+        run: isort --recursive --check-only
+      - name: Flake8
```

Comment:
Nitpick would run pytest in the last step(assuming it takes the longest time) since I prefer to fail earlier.